phalcon/db/adapter/pdo.zep

```diff
@@ -745,7 +762,14 @@ abstract class Pdo extends Adapter
 	 */
 	public function getInternalHandler() -> <\Pdo>
 	{
-		return this->_pdo;
+		var pdo;
+		let pdo = this->_pdo;
+		if typeof pdo != "object" {
+			error_log(typeof pdo);
```

Comment:
Hey!  So this is only passing on PHP 7, which is my environment.  What could this be to have it fail on 5.4-5.6?